src/crepe/api/Config.h

```diff
@@ -32,7 +33,7 @@ class Config final {
 		 *
 		 * Only messages with equal or higher priority than this value will be logged.
 		 */
-		Log::Level level = Log::Level::INFO;
+		Log::Level level = Log::Level::DEBUG;
```

Comment:
Vergeet niet de log level terug te zetten naar wat het in master is. dus van debug naar info